src/test/regress/bin/diff

```diff
@@ -19,12 +19,18 @@ test=$(basename "$file1" .out | sed -E "s/_[0-9]+$//")
 args=${@:1:$#-2}
 BASEDIR=$(dirname "$0")
 
-# whereis searches for standard unix places before $PATH. So select the first
-# entry as the original diff tool.
-DIFF=$(whereis diff | sed "s/diff://g" | awk '{print $1}')
-if [ -z "$DIFF" ]
+DIFF=/usr/bin/diff
+if ! [ -f "$DIFF" ]
```

Comment:
this works fine on OSX. But, do you want to get a review from someone who has more ideas on this change ?